packages/cuba-react-core/package.json

```diff
@@ -40,7 +40,7 @@
     "tslint-config-prettier": "^1.18.0",
     "tslint-react": "^4.0.0",
     "typedoc": "^0.15.3",
-    "typescript": "~3.7.4"
+    "typescript": "~4.1.x"
```

Comment:
Nitpick: I'd use either `~` or `.x` but not both as they have the same meaning in this case.

Reply:
> @karandashovrz CI tests fail, probably there is a breaking change in the new TS that results in compile error in front-generator:
> 
> ```
> src/generators/sdk/model/entities-generation.ts(191,30): error TS2345: Argument of type 'SyntaxKind.NullKeyword' is not assignable to parameter of type 'KeywordTypeSyntaxKind'.
> ```
> 
> Please look into this. Thanks.

@vyacheslav-pushkin fixed

Reply:
> Nitpick: I'd use either `~` or `.x` but not both as they have the same meaning in this case.

@vyacheslav-pushkin fixed